internal/cgroups/cgroups.go

```diff
@@ -104,7 +104,9 @@ func NewCGroups(procPathMountInfo, procPathCGroup string) (CGroups, error) {
 			if err != nil {
 				return err
 			}
-			cgroups[opt] = NewCGroup(cgroupPath)
+			if strings.HasPrefix(cgroupPath, "/sys") {
```

Comment:
If we are not sure that `/sys` is the only point point on all the OSs, should we maybe start with excluding cgroup paths that are known to be wrong instead?

Reply:
Ok, sounds good. Thanks for confirming